velox/exec/OutputBuffer.cpp

```diff
@@ -640,4 +654,46 @@ bool OutputBuffer::isOverutilized() const {
   return (totalSize_ > maxSize_) && !atEnd_;
 }
 
+OutputBufferStats OutputBuffer::getStats() {
+  OutputBufferStats stats;
+  {
+    std::lock_guard<std::mutex> l(mutex_);
+    stats.type = PartitionedOutputNode::kindString(kind_);
```

Comment:
Can we use the kind_ enum here? Why do we need to convert it into a string?

Reply:
I was thinking that the string type here would be more easy to understand. But you've made a point here, and I change the string type to the kind enum in latest update.